build_manifest.json

```diff
@@ -197,8 +195,7 @@
       "l1-artifacts",
       "noir-contracts",
       "p2p",
-      "sequencer-client",
-      "world-state"
```

Comment:
Why this change? Shouldn't we re-run e2e tests if the sequencer changed?

Reply:
maybe we can add them in `rebuildPatterns` instead so they don't have to be installed unnecessarily when e2e tests run but they do signal that they need to run on CI

Reply:
I think should be fine shouldn't it. The `query_manifest` script evaluates dependencies recursively. So even after the change:

```
./scripts/query_manifest dependencies end-to-end                                                                       
acir-simulator
archiver
aztec-node
aztec-rpc
aztec.js
circuits.js
ethereum
foundation
key-store
l1-artifacts
merkle-tree
noir-contracts
p2p
sequencer-client
types
world-state
```

Reply:
Same for rebuild patterns.

```
./scripts/query_manifest rebuildPatterns end-to-end                                                                 
^circuits/
^l1-contracts/
^yarn-project/acir-simulator/
^yarn-project/archiver/
^yarn-project/aztec-node/
^yarn-project/aztec-rpc/
^yarn-project/aztec.js/
^yarn-project/circuits.js/
^yarn-project/end-to-end/
^yarn-project/ethereum/
^yarn-project/foundation/
^yarn-project/key-store/
^yarn-project/l1-artifacts/
^yarn-project/merkle-tree/
^yarn-project/noir-contracts/
^yarn-project/p2p/
^yarn-project/sequencer-client/
^yarn-project/types/
^yarn-project/world-state/
```